README.md

```diff
@@ -45,14 +44,24 @@ This ephys element features automatic ingestion for spike sorting results from t
     + ***WaveformSet*** - A set of spike waveforms for units from a given CuratedClustering
 
 ## Installation
-```
-pip install element-array-ephys
-```
-
-If you already have an older version of ***element-array-ephys*** installed using `pip`, upgrade with
-```
-pip install --upgrade element-array-ephys
-```
+
++ Install `element-array-ephys`
+    ```
+    pip install element-array-ephys
+    ```
+
++ Upgrade `element-array-ephys` previously installed with `pip`
+    ```
+    pip install --upgrade element-array-ephys
+    ```
+
++ Install `element-data-loader`
+
+    + `element-data-loader` is a dependency of `element-array-ephys`, however it is not contained within `requirements.txt`.
```

Comment:
Why not add `element-data-loader` to requirements? If we're moving to the convention of relying on the utilities on multiple root directories across elements, we might avoid this extra installation step by including it as a requirement.

Reply:
Good question @CBroz1 .  A bit of a long explanation.

We have configured `element-data-loader` to allow for installation of optional packages that are not on PyPI, such as CaImAn and scanreader.  This is done using the `extras_require` feature in [setup.py](https://github.com/datajoint/element-data-loader/blob/main/setup.py#L36).  The user would follow these [installation instructions](https://github.com/datajoint/element-data-loader#installation).

Subsequently, from what I understand about PyPI, if a repository has dependencies in `requirements.txt` or `setup.py` that are not on PyPI, then that repository cannot be published to PyPI.  Thus in this case `element-data-loader` cannot be published to PyPI.  And likewise, if we put `element-data-loader` within the `requirements.txt` of `element-array-ephys`, then `element-array-ephys` also would not be able to be published to PyPI.

Given our updated strategy with `element-data-loader` to store the common utility functions among the `elements` and to employ the `extras_require` feature as described above, we have debated not publishing the `elements` on PyPI.  (Note: `element-data-loader` will likely be renamed in the future.)

Reply:
Thanks for the explanation!